app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">HWOne</string>
+    <string name="settings">Settings</string>
+    <string name="log_out">Log out</string>
+    <string name="view_my_contacts">VIEW MY CONTACTS</string>
+    <string name="edit_profile">Edit profile</string>
+    <string name="first_name_sure_name">Lucile Alvarado</string>
+    <string name="career">Graphic designer</string>
+    <string name="home_address">5295 Gaylord Walks Apk. 110</string>
+    <string name="facebook_description">Facebook</string>
+    <string name="linkedin_description">LinkedIn</string>
+    <string name="instagram_description">Instagram</string>
```

Comment:
It's not necessary, but I find it handy to keep content descriptions in a separate string resource file, give it a thought

Reply:
done